kpops/component_handlers/helm_wrapper/helm.py

```diff
@@ -0,0 +1,230 @@
+        release_name: str,
+        dry_run: bool,
+    ) -> str | None:
+        """
+        Prepares and executes the helm uninstall command
+        """
+        bash_command = [
+            "helm",
+            "uninstall",
+            release_name,
+            "--namespace",
+            namespace,
+        ]
+        if dry_run:
+            bash_command.append("--dry-run")
+        try:
+            return self.__execute(bash_command)
+        except ReleaseNotFoundException:
+            log.warning(
+                f"Release with name {release_name} not found. Could not uninstall app."
+            )
+
+    def get_manifest(self, release_name: str, namespace: str) -> Iterable[HelmTemplate]:
+        bash_command = [
+            "helm",
+            "get",
+            "manifest",
+            release_name,
+            "--namespace",
+            namespace,
+        ]
+
+        try:
+            stdout = self.__execute(command=bash_command)
+            return Helm.load_helm_manifest(stdout)
+        except ReleaseNotFoundException:
+            return ()
+
+    @staticmethod
+    def load_helm_manifest(yaml_contents: str) -> Iterator[HelmTemplate]:
+        is_beginning: bool = False
+        template_name = None
+        current_yaml_doc: list[str] = []
+        for line in YamlReader(yaml_contents):
+            if line.startswith("---"):
+                is_beginning = True
+                if template_name and current_yaml_doc:
+                    yield HelmTemplate.load(template_name, "\n".join(current_yaml_doc))
+                    template_name = None
+                    current_yaml_doc.clear()
+            elif is_beginning:
+                template_name = HelmTemplate.parse_source(line)
+                is_beginning = False
+            else:
+                current_yaml_doc.append(line)
+
+    @staticmethod
+    def __extend_tls_config(
+        bash_command: list[str], repo_auth_flags: RepoAuthFlags
+    ) -> list[str]:
+        if repo_auth_flags.ca_file:
+            bash_command.extend(["--ca-file", str(repo_auth_flags.ca_file)])
+        if repo_auth_flags.insecure_skip_tls_verify:
+            bash_command.append("--insecure-skip-tls-verify")
+        return bash_command
+
+    @staticmethod
+    def __enrich_upgrade_install_command(
+        bash_command: list[str],
+        dry_run: bool,
+        helm_command_config: HelmUpgradeInstallFlags,
+    ) -> list[str]:
+        if dry_run:
+            bash_command.append("--dry-run")
+        if helm_command_config.force:
+            bash_command.append("--force")
+        if helm_command_config.wait:
+            bash_command.append("--wait")
+        if helm_command_config.wait_for_jobs:
+            bash_command.append("--wait-for-jobs")
+        if helm_command_config.version:
+            bash_command.extend(["--version", helm_command_config.version])
+        return bash_command
+
+    def __execute(self, command: list[str]) -> str:
```

Comment:
there's a mix of `command` and `bash_command` in this class which is a bit inconsistent. Let's decide on one